pages/services/index.js

```diff
@@ -0,0 +1,101 @@
+import { Text, CtaLink, Testimonial } from "@bridge-the-gap/design-system";
+import { NextSeo } from "next-seo";
+import Link from "@components/link";
+
+export const TestimonialTitle = (
+  <Text variant="3Xl" as="h2">
+    People Say
+  </Text>
+);
+
+export const testimonials = [
+  {
+    name: "Junaid Rasheed",
+    avatar: "/sampleImages/rasheed_avatar.png",
+    text: "Varya is an expert in Design System domain. I have had chance to work with her in past and I found her very intellectual and hardworking individual. She has sounds technical background and deep understanding of web applications development. She leans into solving a problem not just defining it well. I would gladly recommend Varya to anyone who is looking for a Design System expert.",
```

Comment:
We have the same text for all the testimonials. That was OK for the Storybook where it was just an example. But this is a website with real content. We cannot release exemplary texts, they should be real. In the old version of the website, this content was at the "Workshop" page: https://goright.io/hands-on-workshop/